cypress/e2e/tests/administration/repository/subversion.test.ts

```diff
@@ -47,6 +49,8 @@ describe(["@tier1"], "Test secure and insecure svn repository analysis", () => {
     });
 
     beforeEach("Load data", function () {
+        preservecookies();
```

Comment:
We are not making preservecookies() calls anymore . So, this needs to be removed.